src/Microsoft.ML.StandardLearners/Standard/Online/AveragedPerceptron.cs

```diff
@@ -41,12 +41,21 @@ public sealed class AveragedPerceptronTrainer : AveragedLinearTrainer<BinaryPred
 
         public sealed class Options : AveragedLinearArguments
         {
+            /// <summary>
+            /// The custom <a href="tmpurl_loss">loss</a>.
```

Comment:
>tmpurl_loss [](start = 36, length = 11)

What's the story with this tmpurl?

Reply:
Shahab is working on a concepts doc. Those tmpUrl are placeholders for referencing those concepts. 

---
In reply to: [256202773](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256202773) [](ancestors = 256202773)

Reply:
please see [https://github.com/dotnet/machinelearning/issues/2356](https://github.com/dotnet/machinelearning/issues/2356)

---
In reply to: [256251776](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256251776) [](ancestors = 256251776,256202773)